core/client/app/components/gh-profile-image.js

```diff
@@ -0,0 +1,86 @@
+/* global md5 */
+import Ember from 'ember';
+import {request as ajax} from 'ic-ajax';
+
+export default Ember.Component.extend({
+    email: '',
+    size: 90,
+    image: null,
+
+    ghostPaths: Ember.inject.service('ghost-paths'),
+
+    hasUploadedImage: Ember.computed.notEmpty('data'),
+
+    defaultImage: Ember.computed('ghostPaths', function () {
+        var url = this.get('ghostPaths.url').asset('/shared/img/user-image.png');
+        return `background-image: url(${url})`.htmlSafe();
+    }),
+
+    imageBackground: Ember.computed('email', function () {
+        var email = this.get('email') || '',
+            size = this.get('size'),
+            url = 'http://www.gravatar.com/avatar/' + md5(email) + '?s=' + size + '&d=blank';
+        return `background-image: url(${url})`.htmlSafe();
+    }),
+
+    didInsertElement: function () {
+        var self = this,
+            size = this.get('size'),
+            uploadElement = Ember.$('.js-file-input');
+
+        uploadElement.fileupload({
+            url: self.get('ghostPaths.url').api('uploads'),
+            dropZone: Ember.$('.js-img-dropzone'),
+            previewMaxHeight: size,
+            previewMaxWidth: size,
+            previewCrop: true,
+            maxNumberOfFiles: 1,
+            autoUpload: false,
+            acceptFileTypes: /(\.|\/)(gif|jpe?g|png|svg?z)$/i
+        })
+        .on('fileuploadadd', Ember.run.bind(self, self.queueFile))
+        .on('fileuploadprocessalways', Ember.run.bind(self, self.triggerPreview));
+    },
+
+    queueFile: function (e, data) {
+        this.set('data', data);
+    },
+
+    triggerPreview: function (e, data) {
```

Comment:
I don't think so. In all of the examples I saw on the uploader wiki it wasn't used.